flink-connectors/pom.xml

```diff
@@ -43,7 +43,9 @@ under the License.
 		<module>flink-connector-elasticsearch5</module>
 		<module>flink-connector-elasticsearch6</module>
 		<module>flink-connector-elasticsearch7</module>
-		<module>flink-connector-hbase</module>
+		<module>flink-connector-hbase-base</module>
```

Comment:
hint: please also update sql connector module name in the following sql-jar part

Reply:
Updated them as well.